.github/workflows/ci.yml

```diff
@@ -0,0 +1,49 @@
+name: Build salesforce-message-handler
+
+on:
+  pull_request:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
```

Comment:
that's something else to fix up in the repo some time, `master` went out years ago!

Reply:
Yeah, for sure! Seems like this repo got missed. But that's probably best in separate PR.